pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,230 @@
+   <p><strong>Najnowsi redaktorzy i programiści:</strong> <a href="https://www.w3.org/People/kevin/">Kevin White</a>, Eric Velleman, Michel Hansma i Vera Lange. <strong>Opiekun treści:</strong> José Ramón Hilera González. <a href="https://www.w3.org/WAI/ER/tools/acknowledgements">Podziękowania zawierają listę</a> współpracowników i poprzednich redaktorów, w tym <a href="https://www.w3.org/groups/wg/eowg/participants/">uczestników EOWG, </a>Erica Eggerta i Shadi Abou-Zahra.</p>
+   <p>Opracowane przez Grupę Roboczą ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/EO/">EOWG</a>) przy wsparciu <a href="https://www.w3.org/WAI/TIES/">Projektu WAI-TIES</a> w 2006, zaktualizowane przy wsparciu <a href="https://www.w3.org/WAI/ACT/">Projektu WAI-ACT</a> w 2014, i zmienione w 2023 roku w ramach <a href="https://www.w3.org/WAI/about/projects/wai-coop/">Projektu WAI-CooP</a>, współfinansowanego przez Komisję Europejską. </p>  
+   
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Wprowadzenie" class="full" %}
+{:/}
+
+
+Narzędzia oceny dostępności cyfrowej to programy lub usługi internetowe, które pomagają określić, czy treść internetowa spełnia standardy dostępności. W tym dokumencie omówiono cechy narzędzi oceny i inne aspekty, które należy wziąć pod uwagę przy wyborze narzędzi oceny. Łącza do różnych narzędzi znajdują się na [[Liście narzędzi oceny dostępności cyfrowej]](/test-evaluate/tools/list/). Strona oferuje również filtry, które pomagają dokonać wstępnego wyboru, aby znaleźć najbardziej pomocne narzędzie.
+
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Treść strony" class="full" %}
+{:/}
+
+- TOC jest tworzony automatycznie.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Wideo: Omówienie narzędzi oceny {#video}
+
+{% include video-player-data.html
+    yt-id="bn1XJSjc_qM"
+    video-id="tools-for-evaluating"
+%}
+
+_Ten film jest również dostępny na serwerze W3C: [Wideo: Omówienie narzędzi oceny (format pliku: MP4, rozmiar pliku: 51MB)](https://media.w3.org/wai/evaluation-intros/tools-for-evaluating.mp4)._
+
+
+{% include excol.html type="start" id="video-intro-transcript" %}
+
+Transkrypcja oraz opis wizualnej strony filmu
+
+{% include excol.html type="middle" %}
+
+<table>
+  <thead>
+    <tr>
+      <th width="50%">Dźwięk</th>
+      <th>Obraz</th>
+    </tr>
+  </thead>
+<tbody>
+  <tr>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+  </tr>
+  <tr>
+    <td>Istnieją programy i usługi internetowe, które pomogą Ci rozpoznać problemy z dostępnością.</td>
+    <td>Otwiera się skrzynka narzędziowa przy komputerze. Ze skrzynki wychodzi lupa, aby sprawdzić stronę internetową na komputerze. Lupa powiększa słowo „dostępność”.</td>
```

Comment:
@zwiastunsw It seems this description has been reworded. Translations must not change or adapt or add to the meaning of the English version. If you think the original text could benefit from a better wording, feel free to open a dedicated issue so that the editors can consider it.

Do you think of a an alternative to get closer to the original wording?